build/xtask/src/dist.rs

```diff
@@ -439,21 +448,18 @@ pub fn package(
             // We have to cheat a little for (re) generating the
             // srec after signing. The assumption is the binary starts
             // at the beginning of flash.
-            hubtools::binary_to_srec(
-                &cfg.img_file("combined.bin", image_name),
-                "signed",
-                cfg.toml
-                    .memories(image_name)?
-                    .get(&"flash".to_string())
-                    .ok_or_else(|| anyhow!("failed to get flash region"))?
-                    .start,
-                kentry,
-                &cfg.img_file("final.srec", image_name),
-            )?;
-            hubtools::translate_srec_to_other_formats(
-                &cfg.img_dir(image_name),
-                "final",
+            let image_bin =
+                std::fs::read(&cfg.img_file("combined.bin", image_name))?;
+            let start_addr = cfg
+                .toml
+                .memories(image_name)?
+                .get(&"flash".to_string())
+                .ok_or_else(|| anyhow!("failed to get flash region"))?
+                .start;
+            let raw_image = hubtools::RawHubrisImage::from_binary(
```

Comment:
I think the comment above that talks about srec is wrong now?

Reply:
Yup, fixed in 8ad21ecd7562001aa0510fe36746efbdc549e2e4